vMenu/CommonFunctions.cs

```diff
@@ -750,6 +750,19 @@ public static async void KickPlayer(IPlayer player, bool askUserForReason, strin
                 // Kick the player using the specified reason.
                 TriggerServerEvent("vMenu:KickPlayer", player.ServerId, defaultReason);
                 Log($"Attempting to kick player {player.Name} (server id: {player.ServerId}, client id: {player.Handle}).");
+                var dict = new Dictionary<string, string>();
```

Comment:
The way this is added is _very_ messy to be adding it to every function. If you wanted to add discord logging it should be by calling a function and providing a few simple parameters.

You should also not implement this on the client side because the kick action is not even confirmed yet here. It still needs to pass extra permission checks on the server.

There are already kick/ban logging features in vMenu, and you could even listen to the 'confirm' events that happen on the server side in order to log this when the kick/ban actions are confirmed.